cores/arduino/chip.h

```diff
@@ -37,8 +37,7 @@
 #ifdef USBCON
 #include "usb_interface.h"
 #endif //USBCON
-
-#include "ethernet.h"
+#include "callback.h"
```

Comment:
Maybe use variant name to be more precise
variant_callback.h  ?
then 
addVariantCallback()
removeVariantCallback()
callVariantCallback();